src/components/home/components/forms/AddSourceSystemForm.tsx

```diff
@@ -16,34 +19,34 @@ const AddSourceSystemForm = () => {
             <Col>
                 <Row className="mt-2">
                     <Col>
-                        <p className="mb-1"> Source System Name: </p>
+                        <p className={`${styles.formFieldTitle} ms-1 mb-1`}> Source System Name: </p>
```

Comment:
Is there not a more generic way to do this. It seems like a lot of repetition on the styles

Reply:
Well, I've looked at the different methods of using CSS without the need of writing prefixes in classes ourselves, so this came to down to the CSS-modules library, which is automatically included in React. This auto generates a prefix with the name of the CSS file and a random string at the end resulting in: _home_className_randomString_, as a class for the Home page. Before I formatted the classes like: home_className. So basically, for the length of the class in the HTML it does not matter since styles.className is almost as long as home_className, but I understand styles is repetitive.

One thing we could do to resolve this is to import all the loose classes at the top like this:
`import { className1, className2, className3} from 'components/home/home.module.scss'`

But I would rather continue using the styles object, as it is converted to, to indicate it is a styles class. The loose import of classes could also become pretty long. What I also like about the CSS components method is that the styles are imported per component, now I think about, this loose import could be an option for this as the amount of classes are drastically shrunk down when split up by components.

When searching for scoped CSS methods this seemed to be the best. The only improvement I could imagine, and have read about a bit somewhere, is to modify one of React's own environmental files to automatically call on a prefix for CSS files somehow. Would not know how to get this done or where to find it (have searched for it) and if this is a best practice.

Reply:
Sounds like a good choice thanks for the explanation!